packages/esm-patient-banner-app/src/banner/patient-banner.component.tsx

```diff
@@ -3,31 +3,21 @@ import classNames from 'classnames';
 import { useTranslation } from 'react-i18next';
 import { Button, Tag } from '@carbon/react';
 import { ChevronDown, ChevronUp, OverflowMenuVertical } from '@carbon/react/icons';
-import { ExtensionSlot, age, formatDate, parseDate, useConfig, useConnectedExtensions } from '@openmrs/esm-framework';
+import { ExtensionSlot, PatientBannerPatientInfo, useConnectedExtensions } from '@openmrs/esm-framework';
 import ContactDetails from '../contact-details/contact-details.component';
 import CustomOverflowMenuComponent from '../ui-components/overflow-menu.component';
 import styles from './patient-banner.scss';
 
 interface PatientBannerProps {
   patient: fhir.Patient;
   patientUuid: string;
-  onClick?: (patientUuid: string) => void;
-  onTransition?: () => void;
-  hideActionsOverflow?: boolean;
 }
 
-const PatientBanner: React.FC<PatientBannerProps> = ({
-  patient,
-  patientUuid,
-  onClick,
-  onTransition,
```

Comment:
The `onClick` and `onTransition` extension state props are not used in any use of [patient-header-slot on GitHub](https://github.com/search?q=org%3Aopenmrs+patient-header-slot&type=code). I think they are cruft from before we refactored out the patient banner in esm-search.